src/os_win/os_errno.c

```diff
@@ -32,11 +33,28 @@ __wt_map_error_to_windows_error(int error) {
  *	Return a positive integer, a decoded Windows error
  */
 static int
-__wt_map_windows_error_to_error(DWORD winerr) {
+__wt_map_windows_error_to_error(DWORD winerr)
+{
 	return (winerr + windows_error_offset);
 }
 
 /*
+ * __wt_map_error_rdonly --
```

Comment:
@sueloverso I think it'd be nicer to have a generic error mapping function in `os_windows/os_errno.c` than something specific to read only mode. Maybe a translation map similar to: https://hackage.haskell.org/package/base-4.3.0.0/src/cbits/Win32Utils.c

Then I'd expect all APIs in os_windows to translate their error returns into posix versions prior to returning. I'd expect this to replace the current `map_windows_error` methods.